package/network/services/dropbear/Makefile

```diff
@@ -224,7 +224,6 @@ define Package/dropbear/install
 	$(INSTALL_CONF) ./files/dropbear.config $(1)/etc/config/dropbear
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/dropbear.init $(1)/etc/init.d/dropbear
-	$(INSTALL_DIR) $(1)/usr/lib/opkg/info
```

Comment:
If you want to point to a revision for a Fixes tag, it's this one: a377aa9ab534ed8fbbc6bbd867bfd66566da73ce. It clearly forgot to remove this line (but nobody noticed until now)